### PR TITLE
Support loading configuration from file in ee

### LINKED ISSUE #1115: Make configuration module for ensemble evaluator
Should read config from file, and if not present it should yield some sensible default values

### PR DESCRIPTION
**Pull request by DanSava**
**Issue**
Resolves #1115

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

**Comment by sondreso** [comment]
Add to whitelist

File: ert_shared/ensemble_evaluator/context_manager.py [hunk 1]
```diff
@@ -16,38 +16,38 @@
...
     logger.debug("ee ws started")
 
     event_logs = [Path(path.runpath) / "event_log" for path in run_path_list]
     dispatch_thread = Thread(
-        target=_attach_to_dispatch, args=(dispatch_url, event_logs)
+        target=_attach_to_dispatch, args=(ee_config.get("dispatch_url"), event_logs)
     )
     dispatch_thread.start()
 
     # XXX: these magic strings will eventually come from EE itself
```
Comment (jondequinor) [inline ert_shared/ensemble_evaluator/context_manager.py:49]: This `XXX` can now be removed =)

File: ert_shared/ensemble_evaluator/config.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            port = data.get("port", DEFAULT_PORT)
+            return {
+                "host": host,
+                "port": port,
+                "url": f"{host}:{port}",
+                "client_url": f"{host}:{port}/{CLIENT_URI}",
+                "dispatch_url": f"{host}:{port}/{DISPATCH_URI}",
+            }
+        except AttributeError as e:
+            logger.info(
```
Comment (jondequinor) [inline ert_shared/ensemble_evaluator/config.py:39]: Why catch this? Is this the case where I provide a invalid YML config?

I would let this propagate up the stack. Especially during in the development phase.

**Review by sondreso — APPROVED** [review]
LGTM!

**Comment by DanSava** [comment]
I cannot merge it as I was kicked from the team :D

**Pull request merged**